spec/migrations/20181001131632_add_conversion_host_id_to_miq_request_tasks_spec.rb

```diff
@@ -0,0 +1,58 @@
+require_migration
+
+describe AddConversionHostIdToMiqRequestTasks do
+  let(:task_stub) { migration_stub(:MiqRequestTask) }
+  let(:host_stub) { migration_stub(:Host) }
+  let(:conversion_host_stub) { migration_stub(:ConversionHost) }
+
+  migration_context :up do
+    it "when host doesn't exist" do
```

Comment:
Nit, but it would be great if this wrote out what we were testing for.

For example, `it "doesn't set the conversion host when the host object doesn't exist"`

Reply:
Done